notebooks/Solution_Exercises.ipynb

```diff
@@ -0,0 +1,168 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8c3882b9",
+   "metadata": {},
+   "source": [
+    "# Solutions of exercises\n",
+    "\n",
+    "Exercises are available in a few notebooks. Below are solutions to the exercises.\n",
+    "How to use the notebook:\n",
+    "\n",
+    "* **DO NOT RUN THE NOTEBOOK**\n",
+    "* **Copy the code snippet(s) and run the snippet(s) in the relevant notebook**\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "38dce9f9",
+   "metadata": {},
+   "source": [
+    "## Exercises from notebook idr0062_prediction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "75db4ae2",
+   "metadata": {},
+   "source": [
+    "### Exercise 1 "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "47d19053",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Question 1: Read the geojson file:\n",
+    "import json\n",
+    "with open(geojson_file) as f:\n",
+    "    d = json.load(f)\n",
+    "    values = d['features']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e8fa7860",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Question 2: Overlay the shapes from a specific plane on the image\n",
+    "z = 5\n",
+    "fiz, ax = plt.subplots()\n",
```

Comment:
The ``fiz`` is not used anywhere. If a simpler line 

``ax = plt.subplot()\n",``

is used, then the cell still works fine in my hands.